lib/features.js

```diff
@@ -0,0 +1,105 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const hash = require('sha1')
```

Comment:
Agreed; also if we're going to pull in an external module, might as well pull in something like `siphash` which will do the job faster (and with less unnecessary-for-this-use-case crypto-related guarantees) than `sha1`.